src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,83 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import I18n from '../../../i18n';
+import PhoneAccept from '../../../icons/phone.svg';
+import PhoneDecline from '../../../icons/phoneOff.svg';
+import VideoIcon from '../../../icons/video.svg';
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import { parseDate } from '../MessageTime/index';
+import styles from './styles.scss';
+
+// get roomid(DisolayIframe(rid)) and put rid id in iframe link
+// <iframe style="height:80%" src="localhost:3000/livechat/roomId"></iframe>
+// For example, for now I have add link of jitsi and its working fine, similarly we can add link of our video/audio call implemented with webrtc.
+const DisplayIframe = (rid) => (
+	<Screen.Content nopadding>
+		<div className={createClassName(styles, 'iframe')}>
+			<iframe style='height:100%' src='https://meet.jit.si/RocketChatXLKhe6QE6dyRLtTkX'{...rid} />
+		</div>
+	</Screen.Content>
+);
+
+// show call start time function.
+export const ShowCallTime = (props) => (
+	<div className={createClassName(styles, 'callTime')}>
+		<time
+			dateTime={new Date(props.stime).toISOString()}>
+			<div>Call started at {parseDate(props.stime)}</div>
+		</time>
+	</div>
+);
+
+// join call button function.
+export const ShowJoinCallButton = (props) => (
+	<div className={createClassName(styles, 'joinCallBox')}>
+		<div className={createClassName(styles, 'joinMessage')} >
+			<div className={createClassName(styles, 'videoIcon')} >
+				<VideoIcon width={20} height={20} />
+			</div>
+			{I18n.t('Join my room to start the video call')}
+		</div>
+	    <a className={createClassName(styles, 'joinLink')} rel='noreferrer' target='_blank' href={`https://meet.jit.si/RocketChatXLKhe6QE6dyRLtTkX${ props.roomId.rid }`} > <VideoIcon width={20} height={20} /> {I18n.t('Join Call')} </a>
```

Comment:
Why is the following value hard-coded here - `https://meet.jit.si/RocketChatXLKhe6QE6dyRLtTkX` ? The livechat should get the jitsi server info from the core.

Reply:
Hello

Reply:
Thank you